nomad/assets/dashboards/overview.json

```diff
@@ -0,0 +1,867 @@
+{
+    "author_info": {
+        "author_name": "Datadog"
+    },
+    "board_title": "Nomad Overview",
+    "description": "",
```

Comment:
Can we add a description? See examples in our default dashboards. e.g
![image](https://user-images.githubusercontent.com/15065007/85447875-14e30200-b564-11ea-98db-1e0f32e7344a.png)